src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -364,9 +432,7 @@ public JSONObject getDebugState()
         debugState.put("forwardedEndpoints", forwardedEndpointIds.toString());
         debugState.put("trustBwe", Config.trustBwe());
         debugState.put("lastBwe", lastBwe);
-        debugState.put("maxRxFrameHeightPx", maxRxFrameHeightPx);
-        debugState.put("selectedEndpointIds", selectedEndpointIds.toString());
-        debugState.put("pinnedEndpointIds", pinnedEndpointIds.toString());
+        debugState.put("videoConstraints", videoConstraintsMap);
```

Comment:
I don't think this will format correctly when printed, will it?

Reply:
oh, it will https://github.com/jitsi/jitsi-videobridge/pull/1250/commits/df9fef8495802b61b6a98590e8974db8bb0cef48

Reply:
But this is a `Map<String, VideoConstraints>` ?

Reply:
It works (I've tested it) as you'd expect because the root JSONObject knows how to convert the map values to JSON by calling their `toString` method.